docs/templates/schedule.md

```diff
@@ -0,0 +1,40 @@
+# Workspace Scheduling
+
+Templates can configure various aspects of how its workspaces are started and
+stopped, as well as manage the lifecycle of failed or inactive workspaces.
+
+![Schedule screen](../images/template-scheduling.png)
+
+## Schedule
+
+Template admins may define the default values for how long a workspace is
+permitted to run without user activity before it is automatically stopped as
+well as the maximum duration it may remain in a started state before it is
+forcibly stopped.
+
+## User scheduling
+
+For templates where a uniform autostop duration is not appropriate, admins may
+allow users to define their own autostart and autostop schedules.
+
+## Failure Cleanup
+
+Failure cleanup defines how long a workspace is permitted to remain in the
+failed state prior to being automatically stopped. Failure Cleanup is an
+enterprise-only feature.
+
+## Dormancy Threshold
+
+Dormancy Threshold defines how long a workspace is allowed to remain inactive
```

Comment:
Please see comment above about feature names.

Also, suggestion: use active voice in this para. Example: instead of "how long a workspace is allowed to remain..." try "how long Coder allows a workspace to remain..." Customers pay good money for Coder, remind them how hard Coder works for them. :)